eosdis-nasa/earthdata-pub-upload
# removes fs and makes it browser onlyu

# Description

Removes the use of the fs module from upload to allow use in forms.  This makes all components strictly browser only.

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Deploy to sit
3. Confirm ability to upload in dashboard

---

## Change Log

Removes the use of the fs module from upload to allow use in forms.  This makes all components strictly browser only.

* [Add _____](link to commitid)
* [Fix _____](link to commitid)
